core/services/fluxmonitor/flux_monitor.go

```diff
@@ -979,11 +979,35 @@ func (p *PollingDeviationChecker) pollIfEligible(thresholds DeviationThresholds)
 
 	polledAnswer, err := p.fetcher.Fetch(request)
 	if err != nil {
-		logger.Errorw(fmt.Sprintf("can't fetch answer: %v", err), loggerFields...)
+		logger.Errorw("can't fetch answer", append(loggerFields, "err", err)...)
 		p.store.UpsertErrorFor(p.JobID(), "Error polling")
 		return
 	}
 
+	// If the polledAnswer is outside the allowable range, log an error and don't submit.
```

Comment:
nitpick - could be nice to wrap this logic in a helper function since `pollIfEligible()` is already pretty bloated